cloudigrade/account/views.py

```diff
@@ -1,10 +1,39 @@
 """DRF API views for the account app."""
+from django.utils.translation import gettext as _
 from rest_framework import exceptions, mixins, status, viewsets
 from rest_framework.response import Response
+from rest_framework.serializers import ValidationError
 
 from account import serializers
 from account.exceptions import InvalidCloudProviderError
-from account.models import Account, AwsAccount
+from account.models import (Account,
+                            AwsAccount,
+                            Instance)
+
+
+def convert_param_to_int(name, value):
```

Comment:
This isn't a view, perhaps move it to `account.util`?

Reply:
If we plan to use it anywhere else, it's totally cool to move it to an appropriate place in `util.*`, maybe even a new package?